src/test/java/com/amplitude/EventTest.java

```diff
@@ -0,0 +1,45 @@
+package com.amplitude;
```

Comment:
For testing the event, do we need to test we can get the right value from the event object after initiating a new Event?